linetools/isgm/abssystem.py

```diff
@@ -300,6 +300,33 @@ def add_component(self, abscomp, tol=0.2*u.arcsec,
         #
         return test
 
+    def copy(self):
+        """Make a copy of the system
+
+        Returns
+        -------
+        newsys : AbsSystem
+            Copy of current system
+        """
+        newsys = self.__class__(self.coord, self.zabs, self.vlim, zem=self.zem,
+                           abs_type=self.abs_type, NHI=self.NHI,
+                           sig_NHI=self.sig_NHI, flag_NHI=self.flag_NHI,
+                           name=self.name)
+        newsys._components = [comp.copy() for comp in self._components]
+        newsys.kin = self.kin
+        newsys.ZH = self.ZH
+        newsys.sig_ZH = self.sig_ZH
+        newsys.flag_ZH = self.flag_ZH
+        newsys._EW = self._EW
```

Comment:
I wasn't aware of this 'hidden' attribute _EW

Is it new?

Reply:
My feeling is that such attribute it should not exist at the `AbsSystem` level, it's an attribute of `AbsLine` instead